doc/api/datasync.md

```diff
@@ -63,7 +63,17 @@ GET /meteringpoints
 {
     "meteringpoints": [
         {
-            "gsrn": 57131300000000001
+            "gsrn": 57131300000000001,
+            "gridArea": "Tennet",
+            "address":
+                {
+                    "address1": "Dieselstraße 28",
+                    "address2": "3. Stock", omitted if null
+                    "locality": "Niedersachsen", omitted if null
```

Comment:
```suggestion
                    "address2": "3. Stock", // omitted if null
                    "locality": "Niedersachsen", // omitted if null
```

Reply:
It may be required to update the `json` label for this code segment to `jsonc`, if the highlighter does not like comments.